vhd/publishing/new-sku-and-add-image-version.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash -e
+
+required_env_vars=(
```

Comment:
This script looks to be a combination of the two others. Are they still needed or they be removed?

Reply:
I think AKS/agent-baker is still using the other two scripts, I want to confirm with them before removing them (but also don't want block our progress here)

Reply:
@AbelHu @mainred do you need those files or does agent baker have their own copies now?

Reply:
@jsturtevant  @marosset  thanks for the heads-up. We have copies of these files. For building Windows VHD pipeline,  AKS/agentbaker has no dependency on aks-engine except the signed scripts and collect-windows-logs.ps1

Reply:
@mainred It seems like that this file can create sku automatically. I think that we should copy some work from here.